### PR TITLE
Fix inaccurate description in the osu!mania Ranking Criteria

### PR DESCRIPTION
**Pull request by RushFTK**
on osu!mania, its special that Base slider no longer controlled by timing panel, by user setting instead (can press F3/F4 in game to adjusted.)

### CODE REVIEW

**Comment by RushFTK** [comment]
https://osu.ppy.sh/ss/13297596
also can different by this setting.

**Comment by Xinnoh** [comment]
Slider velocity refers to the map's individual changes to scroll speed. Those are the green lines and only affects the difficulty it is on

Scroll speed is the user setting that affects how fast notes go for all maps. This is an entirely separate thing and should not be part of the RC.

**Comment by RushFTK** [comment]
the main purpose to change this is not misleading novice mapper that they think they can adjust scroll speed by setting Slider velocity on timing panel. still use this format of sentence is to keep consist with other RC while suit for reality.

**Comment by RushFTK** [comment]
Reply @Xinnoh 
> Slider velocity refers to the map's individual changes to scroll speed. ... Scroll speed is the user setting that affects how fast notes go for all maps.
on original words, the purpose someone try to setting timing panel is aim to change whole maps scroll speed.

**Comment by RushFTK** [comment]
additional minor things found while on translate, like following:  
- Guidelines should use ‘guideline’ to claimed instead of 'rule'

**Comment by Hydria** [comment]
try "Base slider velocity (1.0x) is based on the user's set scroll speed" instead of "Base slider velocity is based on user settings"

**Comment by Feerum** [comment]
I agree with Hydria's proposal. It would be good to add the (1.0x) to it.

**Comment by Unpredictabl3** [comment]
Can agree with hydria here. Just provides a bit more context here which I think makes the guidelines more informative about base slider velocity.

**Comment by MysticEyes** [comment]
I agree with Hydria's proposal as well, I think that it does a great job of clarifying things. Saying that base slider velocity is based off of user settings makes things a little vague since it doesn't say which setting affects base slider velocity.

File: wiki/Ranking_Criteria/osu!mania/en.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Rule proposals as well as suggestions are discussed in the [Ranking Criteria Sub
 - **Column**: Vertical lines on the playfield where notes are placed. The editor uses `0|1|2|3|4|5|6|7|8|9` for timestamp labels. 
 - **Key mode**: The number of columns present in a difficulty.
 - **Key sounding**: Hit sounding using samples extremely similar or directly from the song in their respective places. This method of hit sounding usually provides low feedback to the player.
-- **Slider velocity**: The speed at which the notes/bar lines move on the playfield. Base slider velocity can be controlled in the timing panel and additional changes can be made through inherited (green) timing points.
+- **Slider velocity**: The speed at which the notes/bar lines move on the playfield. Base slider velocity is based on user settings and additional changes can be made through inherited (green) timing points.
```
Comment (RushFTK) [inline wiki/Ranking_Criteria/osu!mania/en.md]:
```suggestion
- **Slider velocity**: The speed at which the notes/bar lines move on the playfield. Base slider velocity (1.0x) is based on the user's set scroll speeds and additional changes can be made through inherited (green) timing points.
```

**Comment by RushFTK** [comment]
i agreed with yourguys but ... it semmed i delete the whole fork project when in another PR so i couldn't edit this PR's changes anylonger. what should i do ;_;

File: wiki/Ranking_Criteria/osu!mania/en.md (hunk 1 above)
Comment (RushFTK) [inline wiki/Ranking_Criteria/osu!mania/en.md]:
```suggestion
- **Slider velocity**: The speed at which the notes/bar lines move on the playfield. Base slider velocity (1.0x) is based on the user's set scroll speeds and additional changes can be made through inherited (green) timing points.
```

**Comment by RushFTK** [comment]
reopen at #2330 to regain write access to my fork project.

**Pull request closed without merging**